tests/integration/study_data_blueprint/test_link.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import pytest
+from starlette.testclient import TestClient
+
+from antarest.study.storage.rawstudy.model.filesystem.config.links import TransmissionCapacity
+from tests.integration.prepare_proxy import PreparerProxy
+
+
+@pytest.mark.unit_test
+class TestLink:
+    @pytest.mark.parametrize("study_type", ["raw", "variant"])
+    def test_link_update(self, client: TestClient, user_access_token: str, study_type: str) -> None:
+        client.headers = {"Authorization": f"Bearer {user_access_token}"}  # type: ignore
+
+        preparer = PreparerProxy(client, user_access_token)
+        study_id = preparer.create_study("foo", version=820)
+        if study_type == "variant":
+            study_id = preparer.create_variant(study_id, name="Variant 1")
```

Comment:
When the study is a variant it could be nice to check the created `update_link` command to ensure it only contains the data you updated and not all the data